starter-code/src/functions-and-arrays.js

```diff
@@ -11,14 +31,60 @@ var words = [
   'crackpot'
 ];
 
-// Calculating a Sum
 
+// Finding Longest Word
+function findLongestWord(str)
+{
+  var array1 = str.match(/\w[a-z]{0,}/gi);
```

Comment:
You should not need this since on the instructions said that `findLongestWord` accepts an array as argument